app/templates/index.html

```diff
@@ -22,7 +22,7 @@
 								<section id="text-input" style="display:none">
 									<div class="form-group">
 										<label for="textArea"> Enter One Name per Line </label>
-										<textarea class="form-control" type="text" placeholder="Eg. Hong Phuc,Dang,@hpdang,FOSSASIA " id="textArea" name="csv"></textarea>
+										<textarea class="form-control" type="text" placeholder="Eg. Hong Phuc,Dang,Organiser,@hpdang,FOSSASIA " id="textArea" name="csv"></textarea>
```

Comment:
Remove `,` from name `Hong Phuc Dang`

Reply:
@vaibhavsingh97  Name appears in two lines as before.

Reply:
@vaibhavsingh97 Name appears in two lines. You can refer to the issue where Mario explicitly stated the format to be used.